strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/gremlin/adapters/ArtifactIdGroupAdapter.java

```diff
@@ -0,0 +1,160 @@
+{
+
+    @Inject
+    private ArtifactAdapter artifactAdapter;
+
+    @Override
+    public String label()
+    {
+        return Vertices.ARTIFACT_ID_GROUP;
+    }
+
+    public EntityTraversal<Vertex, ArtifactIdGroup> fold(Optional<Class<? extends GenericArtifactCoordinates>> layoutArtifactCoordinatesClass,
+                                                         Optional<ArtifactTag> optionalTag)
+    {
+        EntityTraversal<Vertex, Vertex> artifactsTraversal = optionalTag.map(ArtifactTag::getName)
+                                                                        .map(tagName -> __.outE(Edges.ARTIFACT_GROUP_HAS_TAGGED_ARTIFACTS)
+                                                                                          .has(Properties.TAG_NAME, tagName)
+                                                                                          .inV())
+                                                                        .orElse(__.outE(Edges.ARTIFACT_GROUP_HAS_ARTIFACTS)
+                                                                                  .inV());
+
+        return __.<Vertex, Object>project("id", "uuid", "storageId", "repositoryId", "name", "artifacts")
+                 .by(__.id())
+                 .by(__.enrichPropertyValue("uuid"))
+                 .by(__.enrichPropertyValue("storageId"))
+                 .by(__.enrichPropertyValue("repositoryId"))
+                 .by(__.enrichPropertyValue("name"))
+                 .by(artifactsTraversal.dedup()
+                                       .map(artifactAdapter.fold(layoutArtifactCoordinatesClass))
+                                       .map(EntityTraversalUtils::castToObject)
+                                       .fold())
+                 .map(this::map);
+    }
+
+    @Override
+    public EntityTraversal<Vertex, ArtifactIdGroup> fold()
+    {
+        return fold(Optional.empty(), Optional.empty());
+    }
+
+    private ArtifactIdGroup map(Traverser<Map<String, Object>> t)
+    {
+        ArtifactIdGroupEntity result = new ArtifactIdGroupEntity(extractObject(String.class, t.get().get("storageId")),
+                                                                 extractObject(String.class, t.get().get("repositoryId")),
+                                                                 extractObject(String.class, t.get().get("name")));
+        result.setNativeId(extractObject(Long.class, t.get().get("id")));
+        result.setUuid(extractObject(String.class, t.get().get("uuid")));
+        Collection<Artifact> artifacts = (Collection<Artifact>) t.get().get("artifacts");
+        artifacts.stream().forEach(result::addArtifact);
+
+        return result;
+    }
+
+    @Override
+    public UnfoldEntityTraversal<Vertex, Vertex> unfold(ArtifactIdGroup entity)
+    {
+        String storedArtifactIdGroup = Vertices.ARTIFACT_ID_GROUP + ":" + UUID.randomUUID();
+        EntityTraversal<Vertex, Vertex> connectArtifacstTraversal = __.identity();
```

Comment:
There is a typo in the name of the variable. Could you please fix this here and wherever necessary below? Thanks! :)
```suggestion
        EntityTraversal<Vertex, Vertex> connectArtifactsTraversal = __.identity();
```